preload/ElmSharp.Wearable.preload

```diff
@@ -0,0 +1,10 @@
+###########################################################################################
```

Comment:
I think we can move this directory under src directory and create a simple project file for it which just copies .preload files into Artifacts so that PlatformFileList.txt can also take care of the .preload files.

Reply:
done